src/App.jsx

```diff
@@ -25,21 +26,24 @@ export const alex = {
 
 export const App = () => (
   <div className="App">
-    <section className="Person">
-      <h2 className="Person__name">My name is Misha</h2>
-      <p className="Person__age">I am 37</p>
-      <p className="Person__partner">Natasha is my wife</p>
-    </section>
-
-    <section className="Person">
-      <h2 className="Person__name">My name is Olya</h2>
-      <p className="Person__partner">Maksym is my husband</p>
-    </section>
-
-    <section className="Person">
-      <h2 className="Person__name">My name is Alex</h2>
-      <p className="Person__age">I am 25</p>
-      <p className="Person__partner">I am not married</p>
-    </section>
+    <Person
+      name="Misha"
+      age={37}
+      sex="m"
+      isMarried
+      partnerName="Natasha"
+    />
+    <Person
+      name="Olya"
+      sex="f"
+      isMarried
+      partnerName="Maksym"
+    />
+    <Person
+      name="Alex"
+      age={25}
+      sex="m"
+      isMarried={false}
+    />
```

Comment:
you should only pass person object as a prop
```suggestion
    <Person person={misha} />
    <Person person={olya} />
    <Person person={alex} />
```